### PR TITLE
NT-0004 - Improvement .gitignore

### PR DESCRIPTION
**Pull request by kamilmoskal**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,92 @@
-/node_modules
+# dependencies
+node_modules
```
Comment (mras2303) [inline .gitignore]: Is that node_modules rule contains node_modules in client directory?

Comment (kamilmoskal) [reply .gitignore]: Yes, for me add both to ignore

Comment (mras2303) [reply .gitignore]: Approved

**Comment by mras2303** [comment]
Approved

**Pull request merged**